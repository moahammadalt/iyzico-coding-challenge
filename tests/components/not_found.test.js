import React, { Component } from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import NotFound from '../../src/components/not_found';


// Snapshots
describe('<NotFound /> Snapshot test for NotFound component', () => {

  it('Should match the NotFound snapshot', () => {

    const wrapper = shallow(<NotFound />);

    const tree = renderer.create(wrapper).toJSON();

    expect(tree).toMatchSnapshot();
  });

});