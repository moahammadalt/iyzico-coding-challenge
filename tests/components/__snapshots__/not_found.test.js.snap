// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> Snapshot test for NotFound component Should match the NotFound snapshot 1`] = `
<div
  className="text-center c-w"
>
  <h1>
    This path is not found
  </h1>
</div>
`;
